### DISCUSSION TITLE
The game didn't restart automatically for the latest update + weird errors in the Windows event viewer

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#17251] [Q&A] [answered]
For some reason, the game didn't restart automatically. I had to launch it after it closed for the latest update (on Windows 10, from version `2022.306.0` to `2022.314.0`).

Not sure about the cause since the logs seem clean: they show the "before-update" instance.
Here's, for example, the runtime log:
![example](https://user-images.githubusercontent.com/43073074/158212553-d024108b-b22c-44e0-a0b7-d3353d3a97d6.png)

I didn't want to open this discussion until I noticed these two errors in the event viewer (note: also compare the times with the previous instance, adding 1 to the hour to match my time zone in the screenshot above):

1 - ![1](https://user-images.githubusercontent.com/43073074/158213949-0fe4ece1-00b0-4478-9a9e-b692c2f8578e.png)
2 - Quick translation for this one:
```
Application: osu!.dll
Framework version: v4.0.30319
Description: the process was terminated due to an unhandled exception.
Exception information: exception code e0434352, exception address 00007FFC6EA64F69
Stack:
```
 (and it ends after "Stack:")
 
 ![image](https://user-images.githubusercontent.com/43073074/158215297-cc62ae31-75b5-431c-bb4d-2ff516614c85.png)

### (Note that the game works fine after the update. The issue is that it didn't restart automatically after closing)

**Comment by peppy** [comment, marked as answer]
This is fine. We switched updater and it won't restart this once.
